src/components/com_tc/site/views/content/tmpl/default.php

```diff
@@ -34,16 +34,14 @@ function validateform(form)
 			?>
 			<div class="well well-condensed">
 			<div class="">
-				<h1><?php echo $this->termsandconditions->title; ?></h1>
+				<h1 class="center"><?php echo $this->termsandconditions->title; ?></h1>
 				<strong><?php echo JText::_("COM_TC_LATEST_TERMSANDCONDITIONS_VERSION") . $this->termsandconditions->version; ?>&nbsp;&nbsp;
-				<?php echo JText::_("COM_TC_LATEST_TERMSANDCONDITIONS_UPDATED_DATE") .
-				HTMLHelper::_('date', $this->termsandconditions->modified_on, $this->dateFormat, true); ?></strong>
+				</strong>
 			</div>
 			<br>
-			<div class="">
-				<?php
-						echo nl2br($this->termsandconditions->content);
-		} ?>
+			<div class="" style="overflow-y: scroll; height: 250px;">
```

Comment:
@paritshivani Check this on mobile also check better way instead of adding hardcoded height

Reply:
Working mobile view also